src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -83,6 +83,8 @@
 import org.opensearch.client.opensearch.OpenSearchClient;
 import org.opensearch.client.transport.OpenSearchTransport;
 import org.opensearch.client.transport.rest_client.RestClientTransport;
+import org.opensearch.index.reindex.BulkByScrollResponse;
+import org.opensearch.index.reindex.DeleteByQueryRequest;
```

Comment:
Unusual that these are the only imports that aren't `o.o.client.*`.   Any idea why?

Reply:
Delete/UpdateByQueryRequest classes are part of the Reindex module